src/docs/product/relay/options.mdx

```diff
@@ -6,34 +6,42 @@ redirect_from:
   - /meta/relay/options/
 ---
 
-Configuration for Relay is recorded in the file `.relay/config.yml`. To change this location, pass the `--config` option to any Relay command:
+Configuration for Relay is recorded either in the file `.relay/config.yml`, or when noted, can be overridden through an environment variable. To change the config location, pass the `--config` option to any Relay command:
 
 ```shell
 ❯ ./relay run --config /path/to/folder
 ```
 
 All configuration keys are `snake_case`.
 
+## Credentials
+
+Credentials can be initiated as described in the [getting started page](/product/relay/getting-started/), but can be overridden by setting the following environment variables:
+
+- secret key: `"RELAY_SECRET_KEY"`
+- public key: `"RELAY_PUBLIC_KEY"`
+- id: `"RELAY_ID"`
```

Comment:
Starting the sentences with uppercase look better to me, and I don't think we need the `"`s:
```suggestion
- Secret key: `RELAY_SECRET_KEY`.
- Public key: `RELAY_PUBLIC_KEY`.
- ID: `RELAY_ID`.
```